### PR TITLE
HNS error bestiary.

### PR DESCRIPTION
**Pull request by m-kostrzewa**

### CODE REVIEW

File: doc/developer_guide/HNS_error_bestiary.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# HNS errors bestiary
+
+HNS error messages ofter aren't descriptive enough. 
+Here are error messages the team encountered in different contexts, and their meaning.
+
+## 1. HNS Unspecified Error
+
+1. When attempting to create a transparent docker network
+    * when no other transparent docker network exists and
+    * Ethernet adapter to be used by the transparent networks has no IP address or it's invalid.
```
Comment (krdln) [inline doc/developer_guide/HNS_error_bestiary.md:10]: Or creation of another VMSwitch is already in progress

Comment (m-kostrzewa) [reply doc/developer_guide/HNS_error_bestiary.md:10]: @krdln can you approve/disapprove so that we merge it and you can submit new entries?

**Comment by krdln** [comment]
Does this need to be linked in an index somehow?

**Comment by m-kostrzewa** [comment]
@krdln it should be done automatically: https://github.com/Juniper/contrail-windows-docs/pull/6/files

**Review by krdln — APPROVED** [review]

**Pull request merged**